controllers/kubeadmconfig_controller_test.go

```diff
@@ -32,7 +32,7 @@ import (
 	typedcorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/klog/klogr"
 	bootstrapv1 "sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm/api/v1alpha2"
-	"sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm/certs"
+	cluster2 "sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm/internal/cluster"
```

Comment:
yay auto import naming? 😄

Reply:
lol such yay. Renamed to `internalcluster`